controllers/controllers.js

```diff
@@ -1,7 +1,13 @@
-const { selectTopics } = require("../models/models.js");
+const { selectTopics, selectArticles } = require("../models/models.js");
 
 exports.getTopics = (req, res, next) => {
   selectTopics().then((topics) => {
     res.status(200).send({ topics });
   });
 };
+
+exports.getArticles = (req, res, next) => {
```

Comment:
Consider splitting out your controllers into separate files - e.g. topics.controllers.js, articles.controllers.js etc. It will make it easier in the long run when debugging :)